server/src/services/validation/CompilationService.ts

```diff
@@ -12,6 +12,10 @@ export class CompilationService {
     const hre = this._getHRE();
     const { input } = compilationDetails;
 
+    // This makes the compilation up to 10x faster
+    delete (input.settings as any).optimizer;
```

Comment:
What was the final decision on this one?

I thought we were held to keeping it due to changes to semantics?

Reply:
I'm not sure, do we have a counter example for that ? In my opinion this optimization is a huge QoL booster. The delay between typing and getting diagnostics is very noticeable.